README.md

```diff
@@ -1 +1,14 @@
 # MediaMonks - Frontend Coding Standards
+
+## CSS
+
+We use [Sass](https://sass-lang.com/) as css preprocessor.
+
+#### Selectors
+Only use class names for styling, no id's.
+
+#### z-index
+Never use a `z-index` value higher then `4`. If you need more, use the [zindex](https://github.com/mediamonks/seng-scss/blob/master/utils/function/_zindex.scss) function of [seng-scss](https://www.npmjs.com/package/seng-scss).
```

Comment:
Never manage z-indices that relate to items outside of a component with a hardcoded index. Overlays, Sticky Navigations are examples of items that need a z-index in relation to other components on the page. Use a central managed list of variables in combination with the [zindex](https://github.com/mediamonks/seng-scss/blob/master/utils/function/_zindex.scss) function instead.